src/Appium.Net/Appium/Interfaces/IHasPerformanceData.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+
+namespace OpenQA.Selenium.Appium.Interfaces
+{
+    public interface IHasSupportedPerformanceData
+    {
+        /// <summary>
+        /// Returns the information of the system state which is supported to read as like cpu, memory, networ
+        /// </summary>
+        /// <param name="packageName">The package name of the application</param>
+        /// <param name="dataType">The type of system state which wants to read. It should be one of the supported performance data types.</param>
```

Comment:
it makes sense to mention these data types are returned by `GetSupportedPerformanceDataTypes()` call

Reply:
actually, these data types are static: https://github.com/appium/appium-android-driver/blob/c1a8ee54b69e14940d4c2ecb766be7f75235aba2/lib/commands/performance.js#L12

Reply:
done